pkg/handler/project/testdata/update_project_status/200.json

```diff
@@ -0,0 +1,17 @@
+{
+    "data": {
+        "id": "8dc3be2e-19a4-4942-8a79-56db391a0b15",
+        "createdAt": "2022-11-11T18:06:56.362902Z",
+        "updatedAt": "2022-11-14T00:23:24.00668Z",
+        "name": "Fortress",
+        "type": "dwarves",
+        "status": "active",
+        "startDate": "2022-11-01T00:00:00Z",
+        "endDate": null,
+        "members": [],
+        "technicalLeads": [],
+        "accountManager": null,
+        "salePerson": null,
+        "deliveryManager": null
```

Comment:
we dont want to see the data from line 11 -> 15

Reply:
I create new struct to return it